modules/version/ComponentsVersions.js

```diff
@@ -63,23 +63,19 @@ function(node, mucResource, mucJid) {
 
         var version = item.value;
         if (this.versions[componentName] !== version) {
-            if(this.versions[componentName] &&
```

Comment:
> @bgrozev Why did you delete this code?

The code fires an "incompatible versions event" if the version of the XMPP server changes during a conference, regardless of the version. We can't know whether a version is compatible or not without looking at said version.

Since I very much doubt that we actually encounter changes to the XMPP server version during a conference, I assume the code is effectively dead. And since I could not figure out what the actual intent is, I deleted it in order to spare other people the time and confusion.